cves/kernel/CVE-2021-27363.yml

```diff
@@ -474,7 +477,7 @@ CWE_instructions: |
     CWE: ["123", "456"] # this is ok
     CWE: [123, 456]     # also ok
     CWE: 123            # also ok
-CWE:
+CWE: 000
```

Comment:
Not a valid CWE

Reply:
Mine didn't have a CWE, unlike my first one. How do I find that?

Reply:
Figure out what it is - so head over cwe.mitre.org and take a look at these instructions: https://cwe.mitre.org/documents/cwe_usage/guidance.html

Also just searching for keywords that you know can help you find it quickly.